NewHorizons/Components/NHGameOverManager.cs

```diff
@@ -134,5 +140,59 @@ private void LoadCreditsScene(GameOverModule gameOver)
                     break;
             }
         }
+
+        private void LoadCustomCreditsScene(GameOverModule gameOver, IModBehaviour mod)
+        {
+            var fromScene = LoadManager.GetCurrentScene();
+            var toScene = OWScene.Credits_Fast;
+
+            LoadManager.LoadScene(toScene, LoadManager.FadeType.ToBlack);
+
+            // We need to do this so we can unsubscribe from within the lambda.
+            LoadManager.SceneLoadEvent completeCreditsLoad = null;
+
+            completeCreditsLoad = (fromScene, toScene) =>
+            {
+                // Patch new music clip
+                var musicSource = Locator.FindObjectsOfType<OWAudioSource>().Where(x => x.name == "AudioSource").Single();
```

Comment:
This source should either be one that we create ourself or the exact same audio source that is normally used for the credits. Just grabbing the first one we can find could break too easily I think